src/originWhitelist.js

```diff
@@ -13,7 +13,7 @@ const {URL} = require('url');
 // https://test-studio.code.org                     Test
 // https://dashboard-adhoc-my-branch.cdn-code.org   Ad-Hoc servers
 // http://localhost-studio.code.org:3000            Local development
-const CODE_ORG_URL = /^https?:\/\/(?:[\w\d-]+\.)?(?:cdn-)?code\.org(?::\d+)?$/i;
+const CODE_ORG_URL = /^http(?:s)?:\/\/(?:[\w\d-]+\.)?(?:cdn-)?code\.org(?::\d+)?$/i;
```

Comment:
Why the regex change?  I think these are functionally equivalent.

Reply:
D'oh, that was a mistake, I was having trouble for reasons unrelated to the regex and was clumsily messing with it, forgot to undo.